compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/CompilerDriver.java

```diff
@@ -134,6 +134,9 @@ void loadUtilsPackage() {
     }
 
     void loadLangModules(List<PackageID> pkgIdList) {
+        if (symbolTable.isLangLibLoaded) {
```

Comment:
Shall we move this logic to Compiler.compilePackages ?

Reply:
Please refer
https://github.com/ballerina-platform/ballerina-lang/pull/18139